app/views/Layout.scala.html

```diff
@@ -63,8 +63,8 @@
  }
 
  @additionalHead = {
-        <link href='@controllers.routes.Assets.versioned("stylesheets/application.css")'
-              media="screen" rel="stylesheet" type="text/css"/>
+        <script @{CSPNonce.attr} src='@controllers.routes.Assets.versioned("javascripts/main.js")'></script>
```

Comment:
I think this is over indented compared to some of the other PR's you're doing for this ticket as well, might be worth being it inline with the others